doc/manual/utilities.rst

```diff
@@ -31,83 +49,54 @@ Flashing/Loading tool
 .. argparse::
    :ref: artiq.frontend.artiq_flash.get_argparser
    :prog: artiq_flash
+   :nodefault: 
 
 .. _core-device-management-tool:
 
 Core device management tool
 ---------------------------
 
-The artiq_coremgmt utility gives remote access to the core device logs, the :ref:`core-device-flash-storage`, and other management functions.
+The core management utility gives remote access to the core device logs, the :ref:`core-device-flash-storage`, and other management functions.
 
 To use this tool, it is necessary to specify the IP address your core device can be contacted at. If no option is used, the utility will assume there is a file named ``device_db.py`` in the current directory containing the device database; otherwise, a device database file can be provided with ``--device-db`` or an address directly with ``--device`` (see also below).
 
-To read core device logs::
-
-    $ artiq_coremgmt log
```

Comment:
The command is already mentioned in both 'Installing' and 'RTIO', and the same information is provided by the auto-generated documentation a few lines below this. (Separately, it might be nice to have a dedicated section on the various logs and their management in 'Core device', which could go into more depth, but that'd be a different PR). If we prefer I can add it back, but none of the other utilities commands have their examples or guides on the reference page, so it seems a little out of place.